02-chapter_of_course.Rmd

```diff
@@ -124,6 +124,10 @@ Yet again you can use a link to a website like so:
 
 [A Website](https://yihui.org)
 
+You might want to have users open a website in a new tab by default, especially if they need to reference both the course and a resource at once.
+
+[A Website](https://yihui.org){target="_blank"}
```

Comment:
I’ve had mixed success with this format. Generally had to just use the full html format. Do we know this kind of markdown with an option works in leanpub?

Reply:
yep! I had that issue before, but it seems to be working now in both the preview here and in Cluster 101.